upf/upf_pfcp.c

```diff
@@ -245,6 +249,12 @@ vnet_upf_delete_nwi_if (u8 * name, u32 * sw_if_idx)
 
   vec_add1 (gtm->free_nwi_hw_if_indices, nwi->hw_if_index);
 
+  /*if (nwi->upip_index != UPF_INVALID_UPIP_INDEX)
+    {
+      upf_upip_res_t *ip_res = pool_elt_at_index (gtm->upip_res, nwi->upip_index);
+      pool_put (gtm->upip_res, ip_res);
+    }
+  */
```

Comment:
Please remove commented out code

Reply:
woops
that's a good hint, commented code is actually needed
forgot to bring it back during the test of NWI deletion